### DISCUSSION TITLE
访问域名方式+端口一直都会报错：frp-box.top 发送了无效的响应。

### DISCUSSION THREAD

**Discussion by zzdywc** [natfrp/wiki#321] [使用问题]
如题：
![image](https://github.com/natfrp/wiki/assets/50517455/05d18485-9dcc-49fc-8e3e-05d87070ec76)
只有访问ip+端口的形式才能操作访问验证。

**Comment by chery-5** [comment]
我也遇到了这个问题，请问是怎么解决的？
